src/main/java/appeng/client/render/cablebus/CableBusModel.java

```diff
@@ -75,7 +74,7 @@ public IBakedModel bake( IModelState state, VertexFormat format, Function<Resour
 		Map<ResourceLocation, IBakedModel> partModels = this.loadPartModels( state, format, bakedTextureGetter );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "partModels" which hides the field declared at line 49. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AHiddenFieldCheck)